packages/runtime/src/components/_internal/Space.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { ReactNode } from 'react';
+import { css, CSSObject } from '@emotion/css';
+import { Static, Type } from '@sinclair/typebox';
+import { StringUnion } from '../../utils/stringUnion';
+
+export const SpacePropertySchema = Type.Object({
+  align: StringUnion(['start', 'end', 'center', 'baseline'],{
+    title:'Align',
+    category:'Layout'
+  }),
+  direction: StringUnion(['vertical', 'horizontal'],{
+    title:'Direction',
+    category:'Layout'
+  }),
+  justifyContent: StringUnion([
+    'flex-start',
+    'flex-end',
+    'center',
+    'space-between',
+    'space-around',
+  ],{
+    title:'Justify Content',
+    category:'Layout'
+  }),
+  size: Type.Union([StringUnion(['mini', 'small', 'middle', 'large']), Type.Number()],{
```

Comment:
I think just number type is enought? We don't have 'mini', 'small', 'middle', 'large' in core lib.